config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -277,11 +297,12 @@ presubmits:
         # value: "y"
         resources:
           limits:
+            # Using 6 CPU to speed up bazel build phase
             cpu: 6
-            memory: 24Gi
+            memory: 8Gi
```

Comment:
Given you set big-performance to 16, we should set it to 16 here too (or to 8 there). Those two should be consistent.

Reply:
Done.